resources/documentation/stories.rst

```diff
@@ -0,0 +1,467 @@
+=======
+Stories
+=======
+
+This part contains few stories to help you face common tasks / issues seen
+while using the stack.
+
+Management and main configuration
+=================================
+
+Versioning the inventory
+------------------------
+
+It may be a good idea to version the whole /etc/bluebanquise/inventory/
+folder, to keep track of all changes made, and be able to revert in case of
+issues.
+
+.. note::
+  If used, it may be also a good idea to version the
+  /etc/bluebanquise/playbooks/ and /etc/bluebanquise/roles/custom/ folders.
+
+It may even be better to use an external to the cluster git platform (gitlab,
```

Comment:
```suggestion
It may even be better to use an external git platform (gitlab,
```

If we keep this proposal (push to external), can you please add a note about security: don't upload your inventory to public repositories as it may contain sensitive information about your infrastructure.

Reply:
Yes, solved.